src/qri/dataset.ts

```diff
@@ -404,14 +404,14 @@ export function scriptFromTransform(t: Transform): string {
 
 export interface Readme extends Component {
   scriptPath: string
-  script: string
+  text: string
 }
```

Comment:
👍 backend folks really need to start warning FE teams about breaking changes like this